tools/wptrunner/wptrunner/wptlogging.py

```diff
@@ -65,3 +65,18 @@ def __call__(self, data):
             not self.has_log and
             log_levels[data["level"]] <= self.min_level):
             self.has_log = True
+
+
+__tc_logger = None
+def get_taskcluster_logger(kwargs):
```

Comment:
Using the logging library here seems pretty sketchy, since in theory we're trying to catch logging output are redirect it to mozlog. I think for this case I'd rather just have a custom object to implement writing to the taskcluster markdown file; using logging doesn't really solve much since we don't depend on any of the features of logging.